File: LLM-Translations/GPT-4/codenet/Java/Python/s799002881.py
import sys
import math

def main():
    n, d = map(int, input().split())
    cnt = 0

    for _ in range(n):
        x, y = map(int, input().split())
        if math.sqrt(x*x + y*y) <= d:
            cnt += 1

    print(cnt)

if __name__ == "__main__":
    main()

File: LLM-Translations/GPT-4/codenet/Java/C/s799002881.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <math.h>

int main() {
    int n, d;
    scanf("%d %d", &n, &d);
    int cnt = 0;

    for(int i = 0; i < n; i++) {
        double x, y;
        scanf("%lf %lf", &x, &y);
        if(sqrt(x * x + y * y) <= d) {
            cnt++;
        }
    }
    printf("%d\n", cnt);
    return 0;
}
// 
